back/src/index.js

```diff
@@ -119,6 +121,59 @@ app.get('/comments', (_, res) => {
   });
 });
 
+app.post('/auth', (req, res) => {
+  console.log(req.body);
+  db.User.findOne({ where: { email: req.body.email } }).then((user) => {
+    if (user && comparePassword(req.body.password, user.password)) {
+      const data = {
+        nickname: user.nickname,
+        email: user.email,
+        authLevel: user.authLevel,
+        biography: user.biography,
+        icon: user.icon,
+        enabled: user.enabled,
+      };
+      jsonwebtoken.sign(data, 'privateKey', { expiresIn: '1h' },
+        (err, token) => {
+          if (err) { console.log(err); }
+          res.send({ token });
+        });
+    } else {
+      res.sendStatus(401);
+    }
+  });
+});
+
+app.post('/register', (req, res) => {
+  console.log(req.body);
```

Comment:
No console.log

Reply:
Resolve in 2131c8dc98da6ec64382e2a1e66af6b64050ae9c